data/json/recipes/recipe_ammo.json

```diff
@@ -2439,7 +2439,7 @@
   "time": 500,
   "reversible": false,
   "autolearn": true,
-  "book_learn": [["recipe_bullets" , 2 ] , [ "manual_shotgun", 2 ]],
+  "book_learn": [["recipe_bullets" , 2 ] , [ "manual_shotgun", 2 ] , [ "mag_muzzle", 2 ]],
```

Comment:
Should be `mag_muzzle 1` considering the magazine is explicitly about that subject